entsoe-util/src/main/java/com/powsybl/entsoe/util/MergedXnodeImpl.java

```diff
@@ -20,32 +20,38 @@
 
     private float xdp; // x divider position 1 -> 2
 
+    private String line1Name;
+
+    private boolean line1Fictitious;
+
     private double xnodeP1;
 
     private double xnodeQ1;
 
+    private float b1dp; // b1 divider position 1 -> 2
+
+    private float g1dp; // g1 divider position 1 -> 2
+
+    private String line2Name;
+
+    private boolean line2Fictitious;
+
     private double xnodeP2;
 
     private double xnodeQ2;
 
-    private String line1Name;
+    private float b2dp; // b2 divider position 1 -> 2
 
-    private String line2Name;
+    private float g2dp; // g2 divider position 1 -> 2
 
     private String code;
 
-    public MergedXnodeImpl(Line line, float rdp, float xdp, double xnodeP1, double xnodeQ1, double xnodeP2, double xnodeQ2,
```

Comment:
Please keep this public constructor for retro-compatibility

Reply:
Done